guides/common/assembly_converting-a-host-to-rhel.adoc

```diff
@@ -0,0 +1,8 @@
+ifdef::context[:parent-context: {context}]
```

Comment:
It looks like you're not overwriting `{context}`, so lines 1+2 and 6-8 are IMHO not necessary.